src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
You are styling the `body` tag directly here, which is not recommended according to the checklist. Consider adding a class to the `body` tag and style that class instead.